manual/manual_assertion.rst

```diff
@@ -23,7 +23,7 @@ Projector-based
 
 Projector-based assertion utilises the simple fact that the outcome of a projective measurement can be used to determine if a quantum state is in a specified subspace of the state space.
 The method implemented in pytket transforms an arbitrary projective measurement into measurements on the computational basis [Gushu2020]_. 
-However, this approach is not without limitations. Projectors in general require :math:`n \times n` matrices to represent them; hence it becomes impractical when the size of the asserted subspace is large.
+However, this approach is not without limitations. Projectors in general require :math:`2^{n} \times 2^{n}` matrices to represent them; hence it becomes impractical when the size of the asserted subspace is large.
```

Comment:
Well, OK. (If a "projector" really is just an orthogonal projection onto a subspace of the Hilbert space).

We do need a matrix of this size to represent it as a linear map on a 2^n dimensional Hilbert space.

However a projection COULD instead be represented implicitly as a proper d dimensional vector subspace, represented by an orthonormal basis of d vectors of size 2^n, so needing only d . 2^n complex numbers, which could be a lot better if d is small. However, I suppose anyone who cares will already know this.